src/index.js

```diff
@@ -113,6 +120,11 @@ class Masq {
       this.userId = localStorageUserId
       window.sessionStorage.setItem('userId', this.userId)
     }
+    const localStorageDataEncryptionKey = window.localStorage.getItem('dataEncryptionKey')
+    if (localStorageDataEncryptionKey) {
```

Comment:
Same here, maybe we can directly improt the key if localStorageUserId is defined (in the `if (localStorageUserId)` block)

Reply:
:+1:

Reply:
ok